src/test/fuzz/addrman.cpp

```diff
@@ -85,7 +85,7 @@ class CAddrManDeterministic : public CAddrMan
         // 0, 1, 2, 3 corresponding to 0%, 100%, 50%, 33%
```

Comment:
> Notice that it is adding entries with addr != source, so that lucky `start_bucket` is not playing a role in the benchmark, I guess.

I think it does: The first `Add()` call creates a `CAddrInfo`  entry in `mapInfo` and set `CAddrInfo.source` to the source, no matter if `addr==source` or not. Then `GetNewBucket(uint256, const std::vector<bool>)` which is now called from `MakeTried` uses just that to determine the starting bucket.

> In commit [acf656d](https://github.com/bitcoin/bitcoin/commit/acf656d540a82e6fc30421590305cfe295eabbb5) `fuzz: Use public interface to fill addrman tried tables`, `Add_()` and `Good_()` are not public interface, they are private methods.

That's true - thinking about doing a (fuzz-only) refactoring follow-up, in which this could be switched to `Add()` and `Good()`.

Reply:
Thanks, will fix this in a follow-up!